swarm/swarm.go

```diff
@@ -216,7 +220,7 @@ func (self *Swarm) Stop() error {
 	if self.lstore != nil {
 		self.lstore.DbStore.Close()
 	}
-
+	self.sfs.Stop()
```

Comment:
If the device is busy.. we cannot unmount and this will return after 5 seconds without unmounting. We cannot do much here... the mount will be visible from "mount" command and it can be unmounted using external "umount" command ....